### DISCUSSION TITLE
No grafana dashboards shown

### DISCUSSION THREAD

**Discussion by TheSpirit** [teslamate-org/teslamate#3711] [Getting started] [answered]
Hi,
i just set up teslamate but i can not see and dasboards with grafana. 
Is this due, i do not have a tesla allready (will be here soon) or should i just see something even withou a tesla?
Thank you very much.

**Comment by cwanja** [comment]
You should be able to launch Grafana, but **without** a Tesla vehicle it is expected that there is no data.

**Reply by TheSpirit** [reply]
but i am also unable to see the preconfigured dashboards. is this ok?

**Reply by cwanja** [reply]
Can you share what specifically that means? Do you get an error? Please provide a screenshot as well.

**Reply by TheSpirit** [reply]
sure.
i just thought there are some preconfigure once

<img width="1286" alt="Bildschirmfoto 2024-02-29 um 15 45 18" src="https://github.com/teslamate-org/teslamate/assets/6056918/17f67860-7e10-4064-9f8c-d7770b670969">

<img width="1287" alt="Bildschirmfoto 2024-02-29 um 15 45 35" src="https://github.com/teslamate-org/teslamate/assets/6056918/99039eff-92e1-4e17-8c5f-f4bb8106b523">

**Comment by TheSpirit** [comment, marked as answer]
it works now, i had mixued up the url. sorry for that.
Thank you very much.

**Comment by TheSpirit** [comment]
hab a mixup in the url
